agent/internal/client/mock.go

```diff
@@ -28,6 +28,11 @@ import (
 	"google.golang.org/grpc"
 )
 
+const (
+	MockedGitHubAppToken = "mocked_github_app_token"
```

Comment:
Move these over to `agent/command/git_test.go` and don't export the fields since these constants are used only for that one test suite.